clients/fortnite_api.py

```diff
@@ -15,18 +15,19 @@
 
 ACCOUNT_ID_ADVANCED_LOOKUP_URL = "https://fortniteapi.io/v2/lookup/advanced"
 PLAYER_STATS_BY_SEASON_URL = "https://fortniteapi.io/v1/stats"
-
+RANKED_INFO_LOOKUP_URL = "https://fortniteapi.io/v2/ranked/user"
 
 async def get_player_stats(ctx, player_name, silent):
     """Get player statistics from fortniteapi.io."""
     account_info = await _get_player_account_info(player_name)
 
     player_stats = await _get_player_latest_season_stats(account_info)
+    player_rank = await _get_player_rank(account_info)
```

Comment:
For speed performances if this becomes too slow, we can call this API in parallel with the stats API calls in `_get_player_latest_season_stats` above. Could save about ~1.3 seconds per user search